assistedInstallerService.py

```diff
@@ -115,13 +125,13 @@ def _customized_pod_persistent(self) -> Dict[str, str]:
             logger.error(f"Failed to load yaml: {self.podFile}")
             sys.exit(-1)
 
-        saas_version = "v2.18.4"
+        saas_version = "v2.29.0"
```

Comment:
Looks like this can be removed now?

Reply:
Also, this is just part of #100. Once that merges it won't be part of this PR anymore.

Reply:
Sorry to clarify, can this string be removed? Aren't we setting this with "SAAS_VERSION" now?

Reply:
Moving this to #100

Reply:
Fixed there, thanks!